spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -360,3 +360,212 @@ Another option to start a task using the `task-launcher` would be to create a st
 ```
 http --server.port=9000 | my-task-processor | task-launcher-local
 ```
+
+[[spring-cloud-dataflow-composed-tasks]]
+== Composed Tasks
+
+Spring Cloud Data Flow allows a user to create a directed graph where each node
+of the graph is a task application.  This is done by using the DSL for composed
+tasks.  A composed task can be created via the RESTful API, the Spring Cloud
+Data Flow Shell, or the Spring Cloud Data Flow UI.
+
+=== Creating, Launching, and Destroying a Composed Task
+==== Creating a Composed Task
+The DSL for the composed tasks is used when creating a task definition via the
+task create command. For example:
+```
+dataflow:> app register --name timestamp --type task --uri maven://org.springframework.cloud.task.app:timestamp-task:1.2.0.RELEASE
+dataflow:> app register --name mytaskapp --type task --uri file:///home/tasks/mytask.jar
+dataflow:> task create my-composed-task --definition "mytaskapp && timestamp"
+dataflow:> task launch my-composed-task
+```
+In the example above we assume that the applications to be used by our composed
+task have not been registered yet.  So the first two steps we register two task
+applications.  We then create our composed task definition by using the task
+create command.  The composed task DSL in the example above will, when launched,
+execute mytaskapp and then execute the timestamp application.
+
+But before we launch the my-composed-task definition,  we can view what
+Spring Cloud Data Flow generated for us.  This can be done by executing the
+task list command.
+
+```
+dataflow:>task list
+╔══════════════════════════╤═══════════════════════════════════════════════════════════════
+║        Task Name         │                      Task Definition
+╠══════════════════════════╪═══════════════════════════════════════════════════════════════
+║my-composed-task          │mytaskapp && timestamp
+║my-composed-task-mytaskapp│mytaskapp
+║my-composed-task-timestamp│timestamp
+```
+Spring Cloud Data Flow created three task definitions, one for each of the
+applications that comprises our composed task as well as the composed task
+definition.  We also see that each of the generated
```

Comment:
This is easy for me to get it, but it could be hard for someone to understand it entirely. How about we plug the task-name's in parenthesis? 

Example: 

> ... comprises our composed task (`my-composed-task-mytaskapp` and `my-composed-task-timestamp`) 
> ... as well as the composed task definition (`my-composed-task`)

Or, we could use  "numbers" to indicate the references.